TeamSnapSDK/SDK/DataTypes/TSDKTeam.m

```diff
@@ -193,8 +193,8 @@ - (void)setTimeZone:(NSTimeZone *)timeZone {
 
 - (void)setTimeZoneIanaName:(NSString *)timeZoneIanaName {
     [self setString:timeZoneIanaName forKey:@"time_zone_iana_name"];
-    self.collection.data[@"time_zone"] = timeZoneIanaName;
-    [self.changedValues setObject:[NSNull null] forKey:@"time_zone"];
+    [self setString:timeZoneIanaName forKey:@"time_zone"];
+    [self setChangeValue:[NSNull null] forKey:@"time_zone"];
```

Comment:
We should probably have a `removeChangedValue` to make it more explicit. or a comment.

Reply:
Added